ste/uploader-blobFS.go

```diff
@@ -138,6 +139,11 @@ func (u *blobFSUploader) RemoteFileExists() (bool, error) {
 func (u *blobFSUploader) Prologue(state common.PrologueState) {
 	jptm := u.jptm
 
+	if !thresholdSetup {
```

Comment:
Prologue only runs once.  So, unless I'm missing something, you don't need `thresholdSetup`  Does that answer your question in the comment below?

Reply:
Oh, I thought it was once per file. That's fair.